### ISSUE TITLE
Request for Workaround when No Default Browser

### ISSUE THREAD

**Issue by Ergonomicmike** [5axes/ProfilAnalyser#5]
I have removed my default browser from Windows for security reasons. So, when I click on "Profil Analyze" (should that be 'Profile'?), nothing happens.

Could you add an option for people like me to save the Profiles to an html file? Then I could use my portable web browser to open the files.

(By the way, have you seen WinMerge.org? I assume that your tool is somewhat like theirs.)

**Comment by 5axes** [comment]
Seems to be a very particular issue whose cause is identified and may cause more problems if I try to solve it.  As I cannot test this under all environments I will try to put a message in this case but without guarantee .

**Comment by Ergonomicmike** [comment]
Okay. As a workaround, I'm using your csv export extension with WinMerge. It's a few extra steps for me (export two profiles, open them in WinMerge) and probably not as pretty as the Diff would be in html. But it works.

I'll let you close out this issue.

**Comment by 5axes** [comment]
I would not call it a workaround as the level of information is not the same in the csv file. This solution was the one I was   using before making this plugin (you can also open the csv files in excel) . But if it's ok for you

**Comment by Ergonomicmike** [comment]
Okay, I found another "workaround" that works as you intended. 

I downloaded Vivaldi Portable (web browser). I "installed" it as 'Stand Alone." I told it (twice) to make itself my default (Windows) browser. (It asked during "installation," but that didn't take. I had to go into Settings > General >Start Up and make Vivaldi my default from there.

Once I did that, when I click on your extension, the Vivaldi opens and I see all the pretty presentation.
